fairscale/nn/pipe/async_schedule.py

```diff
@@ -194,13 +194,7 @@ def run_invocation(
         from .multiprocess_pipeline import create_task
```

Comment:
can we move this to the top of the file?

Reply:
This is needed here to avoid a cyclic dependency. Added a TODO to remove the cycle.